### PR TITLE
Improve block spacing in post meta elements

### LINKED ISSUE #238: The space in the post meta is too large
In the post meta, it looks like the theme is using both spaces inside the text blocks, and block spacing on the parent block:
Screen Shot 2022-10-03 at 12 34 24

### PR DESCRIPTION
**Pull request by mikachan**
This PR slightly reduces the block spacing of child elements in the post meta pattern, from 1ch to 0.5ch.

I tried removing the blockGap size altogether, but this meant it inherited the parent block spacing which is much larger. I then tried setting the block spacing on each nested element to 0px, but this meant all the elements had no spacing at all. Reducing the existing spacing seemed like the best option.

Closes https://github.com/WordPress/twentytwentythree/issues/238.

### CODE REVIEW

File: patterns/post-meta.php [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
 		<!-- wp:column {"style":{"spacing":{"blockGap":"0px"}}} -->
 		<div class="wp-block-column">
-			<!-- wp:group {"style":{"spacing":{"blockGap":"1ch"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+			<!-- wp:group {"style":{"spacing":{"blockGap":"0.5ch"}},"layout":{"type":"flex","orientation":"vertical"}} -->
```
Comment (colorful-tones) [inline patterns/post-meta.php:60]:
```suggestion
			<!-- wp:group {"style":{"spacing":{"blockGap":"0.5ch"}},"layout":{"type":"flex"}} -->
```

**Review by colorful-tones — APPROVED** [review]
Tested out the `0.5ch` and looks good. I made one small suggestion for the "Tags:" output to remove `"orientation":"vertical", because this breaks it into two lines.

| before | after |
|---------|------------|
| Screen Shot 2022-10-03 at 5 04 28 PM | Screen Shot 2022-10-03 at 5 04 14 PM |

File: patterns/post-meta.php (hunk 1 above)
Comment (carolinan) [reply patterns/post-meta.php:60]: The two lines are by design:
https://www.figma.com/file/ohYEHkZzGjOse46NceTfj5/Twenty-Twenty-Three?node-id=301%3A441

Comment (colorful-tones) [reply patterns/post-meta.php:60]: I forgot to reference the design. 🤦  Thanks for clarifying @carolinan

**Comment by mikachan** [comment]
Thanks, @colorful-tones!

**Pull request merged**